### PR TITLE
Add integration tests bitbucket

### PR DESCRIPTION
**Pull request by pierrelalanne**
In this PR :  
- we add integration tests for BitBucket
- we clean some logging

Secrets were added to the repo as well.

A successful run of the CI pipeline can be found [here](https://github.com/pierrelalanne/src-fingerprint/runs/5364545846).

### CODE REVIEW

File: tests/integration_test.py [hunk 1]
```diff
@@ -179,8 +185,52 @@ def test_src_fingerprint_no_object_specified(title, cmd_args, expected_output_re
...
+)
+def test_src_fingerprint_bitbucket_object_specified(title, cmd_args, expected_output_repos):
+    output = run_src_fingerprint(
+        provider="bitbucket",
+        token = BITBUCKET_INTEGRATION_TESTS_TOKEN,
+        args=cmd_args+["--provider-url", BITBUCKET_INTEGRATION_TESTS_URL]
+    )
+    output_repos = get_output_repos("fingerprints.jsonl")
+    os.remove("fingerprints.jsonl")
+    assert output_repos == {"src fingerprint test", "main-test-repo"}
```
Comment (Walz) [inline tests/integration_test.py:236]: You don't use `expected_output_repos`
```suggestion
@pytest.mark.parametrize(
    "title, cmd_args, expected_output_repos", [
        (
            "Get all repos accesible to integration tests token for project 'src fingerprint'",
            ["--limit", "10","--object", "src fingerprint"], 
            {"src fingerprint test", "main-test-repo"},
        ),
    ]
)
def test_src_fingerprint_bitbucket_object_specified(title, cmd_args, expected_output_repos):
    output = run_src_fingerprint(
        provider="bitbucket",
        token = BITBUCKET_INTEGRATION_TESTS_TOKEN,
        args=cmd_args+["--provider-url", BITBUCKET_INTEGRATION_TESTS_URL]
    )
    output_repos = get_output_repos("fingerprints.jsonl")
    os.remove("fingerprints.jsonl")
    assert output_repos == expected_output_repos
```

File: tests/integration_test.py [hunk 2]
```diff
@@ -179,8 +185,52 @@ def test_src_fingerprint_no_object_specified(title, cmd_args, expected_output_re
...
+)
+def test_src_fingerprint_bitbucket_no_object_specified(title, cmd_args, expected_output_repos):
+    output = run_src_fingerprint(
+        provider="bitbucket",
+        token = BITBUCKET_INTEGRATION_TESTS_TOKEN,
+        args=cmd_args+["--provider-url", BITBUCKET_INTEGRATION_TESTS_URL]
+    )
+    output_repos = get_output_repos("fingerprints.jsonl")
+    os.remove("fingerprints.jsonl")
+    assert len(output_repos) == 10
```
Comment (Walz) [inline tests/integration_test.py:214]: `expected_output_repos` could be replaced with `number_of_expected_output_repos` to make the test parametrizable.

**Review by Walz — CHANGES_REQUESTED** [review]

**Comment by pierrelalanne** [comment]
Should be good now, successful pipeline [here](https://github.com/pierrelalanne/src-fingerprint/runs/5377277137?check_suite_focus=true).

**Comment by pierrelalanne** [comment]
@Walz, I updated the MR, can you have a look ? Thx

**Review by Walz — APPROVED** [review]

**Pull request merged**